### PR TITLE
CCX-5054: Install CCX on laptop

### PR DESCRIPTION
**Pull request by ram-vibhul**

### CODE REVIEW

**Comment by ram-vibhul** [comment]
> Where's the `Common issues` section or similar?
> 
> This is super basic and does not add anything new to the documentation we are already having. Not even mentioning that this guide will simply not work.
> 
> Have you even tried to follow it @ram-vibhul ? I suggest you do it first, make sure it works and then ask for a review.

@oskar-s9s I made sure to test it in my environment and then i have created docs. Everyones Perspective of documenting stuff is different.  Common Issues are not highlighted. We haven't added this for any document But since you want i can add it.

File: docs/admin/Installation/CCX-Install-Laptop.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+This guide explains how to install Docker Desktop, enable Kubernetes, configure AWS credentials for deploying CCX's datastore in AWS, and deploy CCX on the Kubernetes cluster.
+
+---
+
+## Prerequisites
+
+- A system with **Docker Desktop** installed.
+- AWS credentials available.
+- **Helm** installed on your system. If not, follow the [Helm installation guide](https://helm.sh/docs/intro/install/).
```
[the remark this answers is not in the document]
Comment (ram-vibhul) [reply docs/admin/Installation/CCX-Install-Laptop.md:11]: IMO installation of helm shouldn't be documented

File: docs/admin/Installation/CCX-Install-Laptop.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+# Installing Docker Desktop and Configuring Kubernetes for CCX on Laptop
+
+This guide explains how to install Docker Desktop, enable Kubernetes, configure AWS credentials for deploying CCX's datastore in AWS, and deploy CCX on the Kubernetes cluster.
+
+---
+
+## Prerequisites
+
+- A system with **Docker Desktop** installed.
+- AWS credentials available.
```
[the remark this answers is not in the document]
Comment (ram-vibhul) [reply docs/admin/Installation/CCX-Install-Laptop.md:10]: but these are prerequisites. Shouldn't it be needed

File: docs/admin/Installation/CCX-Install-Laptop.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+- Download Docker Desktop from the [official Docker website](https://www.docker.com/products/docker-desktop/).
+
+
+---
+
+## Step 2: Enable Kubernetes in Docker Desktop
+
+1. Open Docker Desktop and navigate to **Settings > Kubernetes**.
+2. Check the box **Enable Kubernetes** and click **Apply & Restart**.
```
[the remark this answers is not in the document]
Comment (ram-vibhul) [reply docs/admin/Installation/CCX-Install-Laptop.md:25]: ok

**Review by freddielunchbird — APPROVED** [review]
it works fine.

**Pull request merged**